zipkin-web/src/main/resources/templates/v2/layout.mustache

```diff
@@ -26,9 +26,7 @@
         </div>
         <div class="navbar-right" style="width: 50%">
           <div class="navbar-right" style='padding-left: 15px'>
-            <div class="navbar-brand">
-              <span class='muted'>{{environment}}</span>
-            </div>
+            <p class="navbar-text muted" id="environment"></p>
```

Comment:
I changed the markup while I was here... navbar-text is more correct than navbar-brand, according to the Bootstrap docs.